Resources/Documentation/dv_info.md

```diff
@@ -1,32 +1,126 @@
 
-![DV Capture](../dv_vrecord_configuration.jpg "DV Capture") 
+![dv_vrecord_configuration](../dv_vrecord_configuration.jpg)
 
-1. Select the name of the DV device you want to use from the list.
+> **_TIP:_** If no devices are listed, click the "Rescan" button below the list. If this doesn't produce any results, make sure your device is connected and turned on. For additional troubleshooting related to device connectivity, please see the <a href="https://mipops.github.io/dvrescue/sections/troubleshooting.html" target="_blank">Troubleshooting</a> page on the DVRescue documentation website.
 
-1. Specify the playback, sidecar, file naming, recording event and directory options as you normally would for vrecord. For details see [Editing Settings](Resources/Documentation/settings.md)
+3. Select the name of the DV device you want to use from the "Select a DV Device" list. The names correspond with the make and model of the connected decks, as well as a unique device identifier. 
+> **_TIP:_** If you have two decks of the same make and model connected, you will need to determine the corresponding identifier for each (the number listed before the make and model of each deck). To determine the device identifier numbers for two or more decks of the same make and model:
+> - Turn the decks off except one.
+> - Check the name of the device still listed in the vrecord GUI.
+> - Note the device identifier number displayed.
+> - Repeat this for all decks of the same make and model.
+> ![dv_vrecord_device_identifier](../dv_vrecord_device_identifier.jpg)
 
-1. Click "OK" to save.
 
-You can then run passthrough and record modes the same as you would with analog videotape. Please note that if the timecode does not start at the very beginning of the tape, the record mode viewer will not open until the timecode is detected (as soon as the counter starts moving on your deck, the viewer window should pop-up)
+4. Specify the playback, sidecar, file naming, recording event and directory options as you normally would for vrecord. For details see [Editing Settings](Resources/Documentation/settings.md).
 
-When using the GUI, environment parameters can be adjusted in the Config tab.
+5. Click "Save Settings" to save.
 
-![DV Capture config](../dv_vrecord_configuration2.jpg "DV Capture")  
+6. You can then use the passthrough and record modes the same as you would with analog videotape in the "Decklink" tab. Make sure you have a tape inserted before attempting playback or recording. Please see the corresponding sections below for instructions on how to facilitate playback and recording DV tapes.
 
-## Bitstream Error Concealment
+7. When using dvcapture in the vrecord GUI, environment parameters can be adjusted in the "Config" tab.
```

Comment:
I prefer original wording for this line (`When using the GUI, environment parameters can be adjusted in the Config tab.`)